cfme/tests/v2v/test_v2v_manual.py

```diff
@@ -194,3 +194,113 @@ def test_osp_migration_request_details_page_shows_vms_for_not_started_plans():
             3. Details page shows VM
     """
     pass
+
+
+@pytest.mark.tier(2)
+def test_detection_non_uci_vm():
+    """
+    title: Test the non uci vm should not detect while configuring conversion host
+
+    Bugzilla:
+        1810091
+    Polarion:
```

Comment:
Add BZ as with **coverage** marker.
Update other tests too.

Reply:
You haven't made this change @mnadeem92.

Reply:
@mnadeem92 
It should be like this - ```@pytest.mark.meta(coverage=[1810091])```